src/parser/jobs/ast/modules/Horoscope.tsx

```diff
@@ -50,23 +61,48 @@ export default class Horoscope extends Module {
 		this.activations++
 	}
 
+	private onHeliosCast(event: CastEvent) {
+		if (this.cooldowns.getCooldownRemaining(this.data.actions.HOROSCOPE.id) > 0) {
+			return
+		}
+		this.nonHoroscopeHeals++
+	}
+
 	onComplete() {
-		/*
-			SUGGESTION: Didn't activate
-		*/
-		const missedActivations = this.uses - this.activations
-		this.suggestions.add(new TieredSuggestion({
-			icon: this.data.actions.HOROSCOPE_ACTIVATION.icon,
-			content: <Trans id="ast.horoscope.suggestion.expired.content">
-				<ActionLink {...this.data.actions.HOROSCOPE} /> does not activate by itself, so don't forget to use it again or it will expire for no potency.
+
+		if (this.parser.patch.before('5.3')){
+			/*
+				SUGGESTION: Didn't activate
+			*/
+			const missedActivations = this.uses - this.activations
+			this.suggestions.add(new TieredSuggestion({
+				icon: this.data.actions.HOROSCOPE_ACTIVATION.icon,
+				content: <Trans id="ast.horoscope.suggestion.expired.content">
+					<ActionLink {...this.data.actions.HOROSCOPE} /> does not activate by itself, so don't forget to use it again or it will expire for no potency.
 			</Trans>,
-			why: <Trans id="ast.horoscope.suggestion.expired.why">
-				<Plural value={missedActivations} one="# expiration" other="# expirations" />  of Horoscope without reading fortunes again.
+				why: <Trans id="ast.horoscope.suggestion.expired.why">
+					<Plural value={missedActivations} one="# expiration" other="# expirations" />  of Horoscope without reading fortunes again.
 			</Trans>,
-			tiers: SEVERETIES.ACTIVATES_MISSED,
-			value: missedActivations,
-		}))
+				tiers: SEVERITIES.ACTIVATES_MISSED,
+				value: missedActivations,
+			}))
 
+		} else {
+			/*
+				SUGGESTION: AOE heal without horoscope
+			*/
+			this.suggestions.add(new TieredSuggestion({
+				icon: this.data.actions.HOROSCOPE.icon,
+				content: <Trans id="ast.horoscope.suggestion.usage.content">
+					Plan to have <ActionLink {...this.data.actions.HOROSCOPE} /> up before you need to cast <ActionLink {...this.data.actions.HELIOS} /> or <ActionLink {...this.data.actions.ASPECTED_HELIOS} /> .
```

Comment:
While I'm here, just something that bothers me personally (and probably nobody else) - there's a space between the `ASPECTED_HELIOS` link and the period.

Reply:
That makes you and everyone else here. No bulli ;_;